.github/workflows/01-build.yml

```diff
@@ -0,0 +1,18 @@
+name: build-gui
+run-name: Build GUI by ${{ github.actor }}
+
+on: [push,pull_request,workflow_dispatch]
+
+jobs:
+  build-gui:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Linux set up
+        uses: ./.github/actions/linux-setup
+      - name: Make the binary
+        run: make CFLAGS='-O2 -fsanitize=address,undefined'
+      - name: Testing - requires `sudo` for `ptrace()`
+        run: sudo make check VERBOSE=1
+      - name: Install Scanmem
+        run: sudo make install
```

Comment:
Nice touch, I expect it to never break, but may as well get this simple install test done.